cloudspanner/conf/cloudspanner.properties

```diff
@@ -0,0 +1,11 @@
+# YCSB properties.
```

Comment:
Thanks for noticing this, I added the license header. Yes, it is used in the README and more generally provides a set of recommended options, i.e. we'd like that people use this as a base for their own properties in their experiments.